SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +31,129 @@ def customise(process):
 
     return process
 
+
+def customise_SimTkOnly(process):
```

Comment:
@kpedro88 @makortel do you think this can be moved already to era? It can also be done in another cleaning PR..